bin/ck8s

```diff
@@ -36,6 +36,7 @@ usage() {
   echo "  upgrade <wc|sc|both> <vX.Y> apply               runs all apply steps upgrading the environment" 1>&2
   echo "  upgrade <wc|sc|both> <vX.Y> prepare             runs all prepare steps upgrading the configuration" 1>&2
   echo "  validate <wc|sc>                                validates config files" 1>&2
+  echo "  check-tools                                     check locally installed tool versions" 1>&2
```

Comment:
Maybe `check-requirements` would be better, given that the install command is `install-requirements`?